Gateway/Request/RefundDataBuilder.php

```diff
@@ -84,7 +84,7 @@ public function build(array $buildSubject)
         $pspReference = $payment->getCcTransId();
         $orderAmountCurrency = $this->chargedCurrency->getOrderAmountCurrency($payment->getOrder(), false);
         $currency = $orderAmountCurrency->getCurrencyCode();
-        $amount = $orderAmountCurrency->getAmount();
+        $amount = $buildSubjectAmount;
```

Comment:
isn't this going to break the calculation of the ratio for split payments on line 113 (which will now always be 1)?

Reply:
You're right! I'm switching to use the the order total to calculate the ratio

Reply:
Is this fixed now?

Reply:
Yep, fixed in the latest commit